libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/AttributeFilterBody.tsx

```diff
@@ -130,7 +140,7 @@ const AttributeFilterBodyCore: React.FC<IAttributeDropdownBodyExtendedProps> = (
             itemsCount={props.totalCount}
             isLoading={isElementsLoading}
             isLoadingClass={LoadingMask}
-            getItemKey={(item: IAttributeElement) => item?.uri || uuid()}
+            getItemKey={(item: IAttributeElement) => item?.uri || "empty"}
```

Comment:
Can we use something less likely to collide with actual data? On tiger a label may conceivably have the value "empty".